Dockerfile

```diff
@@ -31,6 +29,4 @@ RUN for m in $MODELS; \
 ADD scripts/torchserve_start.sh /usr/local/bin/start.sh
 RUN chmod 777 /usr/local/bin/start.sh
 ENV MODELS ${MODELS}
-CMD ["/usr/local/bin/start.sh", "\"${MODELS}\""]
-
-
+CMD ["/usr/local/bin/start.sh", "\"${MODELS}\""]
```

Comment:
New line should be here. See: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file